### ISSUE TITLE
Information for `raw-dylib` is outdated

### ISSUE THREAD

**Issue by tim-weis** [kennykerr/blog#5]
The section "Understanding the windows-targets crate" ends with

https://github.com/kennykerr/blog/blob/04fcf0004978eba81af4e9f67a8f8440b65cc5ca/src/rust-getting-started/understanding-windows-targets.md?plain=1#L44

> As of this writing, the `raw-dylib` feature is not yet stable.

Is this still accurate? I was under the impression that this [stabilized in 1.71.0](https://blog.rust-lang.org/2023/07/13/Rust-1.71.0.html) (for all target architectures). Does this entry need an update?

**Comment by kennykerr** [comment]
Yes, I think this is mostly stable now. There were some lingering issues but nothing that affects `windows-targets` I believe. Feel free to shoot me a PR to update this. I hope to move this to the windows-rs repo when I get some spare time.
